mtmount/Dockerfile

```diff
@@ -0,0 +1,111 @@
+# This docker file cannot fully automate the EUI installation. You must carry on
+# with the installation of the VIPM package installs to complete the installation. 
+
+FROM centos:centos7
+
+LABEL maintainer Andrew Heyer <aheyer@lsst.org>
+
+ENV USER ${USER:-saluser}
+ENV HOME /home/saluser
+USER root
+# Create user and group
+RUN groupadd lsst && adduser -u 1004 -m -g lsst -s /bin/bash saluser
+
+COPY lsst-ts-nexus.repo /etc/yum.repos.d/lsst-ts-nexus.repo
+COPY wandisco-git.repo /etc/yum.repos.d/wandisco-git.repo
+RUN rpm --import http://opensource.wandisco.com/RPM-GPG-KEY-WANdisco
```

Comment:
Instead of WANDisco, consider IUS repo.

```
sudo yum install -y https://centos7.iuscommunity.org/ius-release.rpm
sudo yum install -y git2u-all
```